static/js/shared/ga_events.test.tsx

```diff
@@ -682,3 +722,400 @@ describe("test ga event tool chart plot", () => {
     });
   });
 });
+describe("test ga event tool stat var click", () => {
+  test("call gtag when a stat var is selected in the stat var hierarchy", async () => {
+    // Mock gtag.
+    const mockgtag = jest.fn();
+    window.gtag = mockgtag;
+    // Mock child stat var groups.
+    axios_mock();
+
+    // When the component is rendered.
+    const statVarHierarchy = render(
+      <StatVarHierarchy {...STAT_VAR_HIERARCHY_PROPS} />
+    );
+    // Wait for the collapsible__trigger to get rendered.
+    await waitFor(() => {
+      expect(
+        statVarHierarchy.container.getElementsByClassName(
+          "Collapsible__trigger"
+        )[0]
+      ).toBeTruthy();
+    });
+
+    // Click collapsible trigger.
+    fireEvent.click(
+      statVarHierarchy.container.getElementsByClassName(
+        "Collapsible__trigger"
+      )[0]
+    );
+    const inputId = "#" + STAT_VAR_3 + "dc\\/g\\/Demographics-" + STAT_VAR_3;
+    // Wait for stat vars to get rendered.
+    await waitFor(() => {
+      expect(statVarHierarchy.container.querySelector(inputId)).toBeTruthy();
+    });
+
+    // Click the checkbox of the stat var.
+    fireEvent.click(statVarHierarchy.container.querySelector(inputId), {
+      target: { checked: true },
+    });
+    await waitFor(() => {
+      // Check gtag is called once.
+      expect(mockgtag.mock.calls.length).toEqual(1);
+      // Check the parameters passed to the gtag.
+      expect(mockgtag.mock.lastCall).toEqual([
+        "event",
+        GA_EVENT_TOOL_STAT_VAR_CLICK,
+        {
+          [GA_PARAM_STAT_VAR]: STAT_VAR_3,
+        },
+      ]);
+    });
+  });
+  test("call gtag when a place is added in the place seaerch bar", async () => {
```

Comment:
curious why you put this test in this "test ga event tool stat var click" section, because it seems like this test is about place?

Reply:
Done. My mistake. It shouldn't be there.